ui/admin/tests/acceptance/users/list-test.js

```diff
@@ -41,17 +40,36 @@ module('Acceptance | users | list', function (hooks) {
       'withChildren'
     );
     instances.user = this.server.create('user', {
-      scope: instances.org,
+      scope: instances.orgScope,
     });
-    urls.users = `/scopes/${instances.org.id}/users`;
+    orgURL = `/scopes/${instances.orgScope.id}`;
+    urls.users = `/scopes/${instances.orgScope.id}/users`;
     authenticateSession({});
   });
 
   test('can navigate to users with proper authorization', async function (assert) {
-    // TODO
+    assert.expect(2);
+    await visit(orgURL);
+    assert.ok(
+      instances.orgScope.authorized_collection_actions.users.includes('list')
+    );
+    assert.ok(find(`[href="${urls.users}"]`));
+  });
+
+  test('User cannot navigate to index without either list or create actions', async function (assert) {
+    assert.expect(2);
+    instances.orgScope.authorized_collection_actions.users = [];
+    await visit(orgURL);
+    assert.notOk(
+      instances.orgScope.authorized_collection_actions.users.includes('list')
+    );
+    assert.notOk(find(`[href="${urls.users}"]`));
   });
 
-  test('cannot navigate to users with proper authorization', async function (assert) {
```

Comment:
Just wondering and asking out of the context, pardon me if I ask something too obvious :sweat_smile: Are we planning to add test where the user can not navigate because has not permissions?

Reply:
ahh. it should be `without`. good catch!

Reply:
we renamed it to `User cannot navigate to index without either list or create actions` on L59

Reply:
ahhh that makes sense!! see, it was an obvious one lol. Terrific job with all the capabilities testing @DhariniJeeva :clap: :clap: